config/scorecard/kustomization.yaml

```diff
@@ -14,3 +14,9 @@ patchesJson6902:
     kind: Configuration
     name: config
 # +kubebuilder:scaffold:patchesJson6902
```

Comment:
I wonder if this marker should be at the end of the patch list? Not sure if it makes any changes?

Reply:
I'm not sure what this is used for. It's generated by the Operator SDK/Kubebuilder. I left it where it was because everything prior is generated YAML.

Reply:
OperatorSDK seems to lack documentations on `# +kubebuilder:scaffold:*` :D I think, as tests are working as expected, it is okay to leave till they release the documentations: https://github.com/kubernetes-sigs/kubebuilder/issues/1487